src/main/charts/common/templates/_jmx.tpl

```diff
@@ -30,6 +30,8 @@ Jmx init container
   image: {{ .Values.monitoring.jmxExporterImageRepo}}:{{ .Values.monitoring.jmxExporterImageTag}}
   command: ["cp"]
   args: ["/opt/bitnami/jmx-exporter/jmx_prometheus_javaagent.jar", "{{ .Values.volumes.sharedHome.mountPath }}"]
+  securityContext:
```

Comment:
That is the nice part of having this in the common helm chart. Only single change.